src/ui-lib/sass/gravity.scss

```diff
@@ -81,3 +82,4 @@
 // === Utilities layer ===
 // Utilities and helper classes with ability to override anything which goes before in the
 // triangle, eg. hide helper class.
+@import '06-utilities/has-centered-text';
```

Comment:
Should we have utility classes grouped by functionality? This one is a very specific file name.

Reply:
fair point, I'll get it a generic `text` file for now